src/strategies/adventureland/index.ts

```diff
@@ -0,0 +1,33 @@
+export const author = 'adventureland';
+export const version = '0.0.1';
+
+interface ApiReturn {
+  staked: number;
+}
+
+const apiUrl = 'https://api.adventureland.finance';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const promisesBalances = addresses.map((address) =>
+    fetch(
+      `${apiUrl}/api/governance/vote/${address}/${blockTag}`
+    ).then((response) => response.json())
+  );
+  const balances: ApiReturn[] = await Promise.all(promisesBalances);
```

Comment:
Hi, sorry i've merged your PR but then reverted it. The reason is that this line will not scale well, if there is 1000 votes there will be 1000 requests being sent, the strategy are process on browser so this wont work well. I recommend to use the existing strategy called "api" this strategy will take multiple address into account. Have a look how is work here: https://github.com/snapshot-labs/snapshot.js/pull/180/files

Reply:
Oh, didn't even look into that strategy. I have got that setup. Thanks for the suggestion!